src/serde/value/owned/se.rs

```diff
@@ -12,24 +16,24 @@ impl Serialize for Value {
         S: ser::Serializer,
     {
         match self {
-            Self::Static(StaticNode::Bool(b)) => serializer.serialize_bool(*b),
-            Self::Static(StaticNode::Null) => serializer.serialize_unit(),
-            Self::Static(StaticNode::F64(f)) => serializer.serialize_f64(*f),
-            Self::Static(StaticNode::U64(i)) => serializer.serialize_u64(*i),
+            Value::Static(StaticNode::Null) => serializer.serialize_unit(),
```

Comment:
Naming. My brain trips on `serialize_unit` when I see it in the context of `null`. Nothing to resolve here though, not sure `serialize_null` is any better as we cannot distinguish `null` value from `null` reference and some uses may imply `undefined` because JSON. Perhaps its right my brain should be trip right here!

Reply:
I agree but sadly not in our control, this naming is dictated by `serde` :(